.github/workflows/main.yml

```diff
@@ -0,0 +1,108 @@
+name: build
+on:
+  push:
+    branches:
+      - main
+      - master
+      - poggenpower*
+  pull_request:
+    branches: 
+      - main
+      # comment
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: ${{ github.repository }}
+
+
+jobs:
+  checkout:
+    name: Checkout
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-PIPELINE-0005   
**Check Name:** persist-credentials is true   
**Severity:** HIGH   
**Message:** Consider adding persist-credentials: false to the checkout action in job 'Checkout' inputs  
**Resolution:** Add persist-credentials: false to the checkout action  
[This comment was created by Aqua Pipeline]